sdk/test/UnitTests/TestXRayOptions.cs

```diff
@@ -201,16 +201,16 @@ public void TestUseRuntimeErrorsTrue()
         }
 
         [TestMethod]
-        public void TestUseRuntimeErrorsDefaultsTrue_WhenNotSpecifiedInJson()
+        public void TestUseLogErrorsDefaultsTrue_WhenNotSpecifiedInJson()
         {
             IConfiguration configuration = TestXRayOptions.BuildConfiguration("DisabledXRayMissing.json");
 
             _xRayOptions = XRayConfiguration.GetXRayOptions(configuration);
 
             AWSXRayRecorder.InitializeInstance(configuration);
 
-            Assert.IsTrue(_xRayOptions.UseRuntimeErrors);
-            Assert.AreEqual(AWSXRayRecorder.Instance.ContextMissingStrategy, Core.Strategies.ContextMissingStrategy.RUNTIME_ERROR);
+            Assert.IsFalse(_xRayOptions.UseRuntimeErrors);
```

Comment:
nit: since the assertion is that `UseRuntimeErrors` is false, the name of the test  could also reflect that `TestUseRuntimeErrorsDefaultsFalse_WhenNotSpecifiedInJson()`

Reply:
ah good catch. Will make this change.